### DISCUSSION TITLE
Coworker's account was suspended without notice

### DISCUSSION THREAD

**Discussion by mlptf2** [community/community#82203] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My coworker's account was suspended without notice -- no warnings, no emails, nothing at all. It looks like it has something to do with his organization being flagged, but we're not sure. He was unable to submit a support ticket initially, but it looks like it finally went through: his support ticket ID is #2499078. Anyway, he wanted me to relay that he is annoyed by how his account was suspended out of nowhere without being given any notice at all despite being on a pro plan. We hope that this can be resolved quickly because he's unable to do any work without his account. Thank you.

Support ticket IDs: #2499078 (his), #2499020 (mine)

**Comment by dgrfps** [comment]
Same here, suddenly my account was flagged with no warning, no notifications, no email, nothing. 
My friend asked me if i deleted my account, because my info isn't appearing on our repositories and my profile returns a 404 page.
When i open GitHub support page it says that my account got flagged but doesn't show why, when i try to search for code it says that i'm flagged.
Support still not answered me.
![Screenshot 2023-12-26 173902](https://github.com/community/community/assets/45692685/23396798-58ff-4d14-afe2-dce21666aa90)
